netmanager/src/views/components/DataDisplay/DeviceView/DeviceDeployStatus.js

```diff
@@ -197,11 +198,25 @@ const DeviceRecentFeedView = ({ recentFeed, runReport }) => {
           >
             <span>
               Device last pushed data{' '}
-              <span className={elapsedDurationSeconds > elapseLimit ? classes.error : classes.root}>
-                {getFirstNDurations(elapsedDurationMapper, 2)}
-              </span>{' '}
-              ago.
+              {isDateInPast(recentFeed.created_at) ? (
+                <>
+                  <span
+                    className={elapsedDurationSeconds > elapseLimit ? classes.error : classes.root}
+                  >
+                    {getFirstNDurations(elapsedDurationMapper, 2)}
+                  </span>{' '}
+                  ago.
+                </>
+              ) : (
+                <span className={classes.error}>in the future.</span>
+              )}
             </span>
+            {!isDateInPast(recentFeed.created_at) && (
+              <div className={classes.error}>
+                Error: Start date for this device is set to{' '}
+                {formatDateString(recentFeed.created_at)}
+              </div>
+            )}
```

Comment:
Informs user that date is in the future

Reply:
Thanks @Codebmk , please share screenshot of how the "future" message would look like?

Reply:
@Baalmart I shared it at the top but resharing here
![Screenshot from 2023-05-12 07-12-57](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/43d5d625-6db2-45c8-b6a6-7da57463a920)